### PR TITLE
remove checkboxes from dt1 widget

### LINKED ISSUE #307: need to merge DT1 sub-tile flags edit/preview UI
Screen Shot 2021-06-06 at 12 45 48 PM

You should only have to click on the grid. The check-boxes are not necessary, and take up a lot of real estate on the screen.

### PR DESCRIPTION
**Pull request by gravestench**
* resolves #307
* adds mouse-click support to the subtile flag display
* highlights hovered subtile with a crosshair

### CODE REVIEW

**Comment by gravestench** [comment]
https://user-images.githubusercontent.com/936440/120951374-86787980-c6fd-11eb-996f-f50be8a5bb2f.mp4

**Review by gucio321 — APPROVED** [review]
very cool.

**Review by gucio321 — CHANGES_REQUESTED** [review]
orientation of subtile flags was changed:
at master:  image
now: image

**Review by gucio321 — DISMISSED** [review]
but after a discussion on Discord, it looks like it is a fix :grinning:

File: hswidget/dt1widget/widget.go [hunk 1]
```diff
@@ -646,24 +630,18 @@ func (p *widget) makeSubTilePreview(tile *d2dt1.Tile, state *widgetState) giu.La
...
+					flagPoint := image.Point{X: p1.X + ox, Y: p1.Y + oy}
 
 					// nolint:gomnd // const
-					col := color.RGBA{
-						R: 0,
-						G: 255,
-						B: 255,
-						A: 255,
-					}
+					col := color.RGBA{R: 0, G: 255, B: 255, A: 255}
```
Comment (gucio321) [inline hswidget/dt1widget/widget.go]: btw we can use `math.MaxUint8` to avoid `nolint`

**Review by gucio321 — DISMISSED** [review]

**Pull request merged**